Blockchain/Homebrew/Exchange/Presenters/ExchangeCreatePresenter.swift

```
@@ -304,6 +294,11 @@ extension ExchangeCreatePresenter: ExchangeCreateOutput {
         interface?.apply(presentationUpdates: [.loadingIndicator(visibility)])
     }
 
+    func exchangeButtonEnabled(_ enabled: Bool) {
+        hideError()
```

If I call `exchangeButtonEnabled` I wouldn't think this would also hide the error label. I know why it should be hidden but, it shouldn't be in this function is what I mean.